app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1115,12 +1115,16 @@ void handleScrollEvent(final int aHandle, final int aDevice, final float aX, fin
             if (!isWidgetInputEnabled(widget)) {
                 return;
             }
-            if (widget != null) {
-                float scrollDirection = mSettings.getScrollDirection() == 0 ? 1.0f : -1.0f;
-                MotionEventGenerator.dispatchScroll(widget, aDevice, true,aX * scrollDirection, aY * scrollDirection);
-            } else {
-                Log.e(LOGTAG, "Failed to find widget for scroll event: " + aHandle);
+            if (widget == null) {
+                if (getNavigationBar().isInVRVideo()) {
```

Comment:
Wouldn't be better to assign the MediaControlsWidget in the initialization of the 'widget' variable ? 
Also, wouldn't be the case that when playing a VR video the widget we get from the handle is going to be null always ? If my  assumption is correct, we could do  something like this:

```
Widget widget = getNavigationBar().isInVRVideo() ? getNavigationBar().getMediaControlsWidget() : mWidgets.get(aHandle);
```

Reply:
Actually, I don't think the change you asked is good, as of the flexibility later. We might add some other logic when the widget is null (not focusing on any widget) while we press the D-pad, such as what is asked here #927
> As a suggestion you could use the trigger buttons on the back in conjunction with the joysticks so that you could use up and down to zoom in and out, left and right to rotate the screen and a button to drag the screen and place it wherever you want for a more comfortable viewing position.